### PR TITLE
remove them

### PR DESCRIPTION
**Pull request by keithk**
## Links
* Remix: https://keith-community.glitch.me/about/company

### CODE REVIEW

File: src/presenters/pages/about/company.js [hunk 1]
```diff
@@ -10,63 +10,75 @@ const AboutCompanyPage = () => (
...
+					Slack and Google Hangouts are the normal way we communicate.
         </p>
       </article>
       <article>
         <Heading tagName="h3">A considered work environment</Heading>
         <p>We pioneered remote-working and private offices for creative workers over a decade ago.</p>
-        <p>Our work environment has been carefully and thoughtfully designed to give each person both private and collaborative workspaces. </p>
+        <p>
+					Our work environment has been carefully and thoughtfully designed to give each person both private
+					and collaborative workspaces.{' '}
```
Comment (whimsicallyson) [inline src/presenters/pages/about/company.js]: I assume prettier did this, but it doesn't seem necessary

File: src/presenters/pages/about/company.js [hunk 2]
```diff
@@ -135,18 +150,6 @@ const AboutCompanyPage = () => (
...
-          <p>Tim Kington</p>
-        </span>
-        <span>
-          <img
-            className={styles.avatar}
-            src="https://cdn.glitch.com/d2b595e6-45a6-4ddc-8110-038cdb509b16%2Fmaurice-cherry-removebg-preview%20(1).png?v=1561988404303"
-            alt=""
-          />
-          <p>Maurice Cherry</p>
-        </span>
```
Comment (whimsicallyson) [inline src/presenters/pages/about/company.js]: do we not want antoinette, keith, jlord?

Comment (keithk) [reply src/presenters/pages/about/company.js]: We do, I'm waiting on some headshots for everyone, just wanted to make this change first

File: src/presenters/pages/about/company.js (hunk 1 above)
Comment (keithk) [reply src/presenters/pages/about/company.js]: ESlint did this I think 😭 I just did a --fix

**Pull request merged**